### PR TITLE
Dịch "deep-rnn_vn" - Phần 3

### LINKED ISSUE #1662: Dịch "deep-rnn_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_recurrent-modern/deep-rnn_vn.md#L148-L199

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1662

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_recurrent-modern/deep-rnn_vn.md [hunk 1]
```diff
@@ -178,8 +181,10 @@ d2l.train_ch8(model, train_iter, vocab, lr, num_epochs, ctx)
 * There exist many different flavors of deep RNNs, such as LSTMs, GRUs, or regular RNNs. Conveniently these models are all available as parts of the `rnn` module in Gluon.
 * Initialization of the models requires care. Overall, deep RNNs require considerable amount of work (such as learning rate and clipping) to ensure proper convergence.
 -->
+* Trong các mạng nơ-ron truy hồi sâu, thông tin trạng thái ẩn được truyền tới bước thời gian kế tiếp ở cùng tầng, và truyền tới bước thời gian hiện tại ở tầng kế tiếp.
```
Comment (duythanhvn) [inline chapter_recurrent-modern/deep-rnn_vn.md:184]:
```suggestion
* Trong các mạng nơ-ron truy hồi sâu, thông tin trạng thái ẩn được truyền tới bước thời gian kế tiếp ở cùng tầng và truyền tới bước thời gian hiện tại ở tầng kế tiếp.
```

File: chapter_recurrent-modern/deep-rnn_vn.md [hunk 2]
```diff
@@ -178,8 +181,10 @@ d2l.train_ch8(model, train_iter, vocab, lr, num_epochs, ctx)
...
+* Có nhiều lựa chọn khác nhau cho mạng RNN sâu, ví dụ LSTM, GRU, hoặc là RNN thông thường. Những mô hình này được lập trình sẵn trong mô-đun `rnn` của Gluon.
```
Comment (duythanhvn) [inline chapter_recurrent-modern/deep-rnn_vn.md:185]:
```suggestion
* Có nhiều lựa chọn khác nhau cho mạng RNN sâu, ví dụ LSTM, GRU hoặc là RNN thông thường. Những mô hình này được lập trình sẵn trong mô-đun `rnn` của Gluon.
```

File: chapter_recurrent-modern/deep-rnn_vn.md [hunk 3]
```diff
@@ -194,7 +199,12 @@ d2l.train_ch8(model, train_iter, vocab, lr, num_epochs, ctx)
 4. Would you want to combine sources of different authors when modeling text? Why is this a good idea? What could go wrong?
 -->
 
-*dịch đoạn phía trên*
+1. Hãy lập trình một mạng RNN hai tầng từ đầu sử dụng mã nguồn cho mạng một tầng chúng ta đã thảo luận ở :numref:`sec_rnn_scratch`.
+2. Hãy thay thế khối LSTM bằng khối GRU và so sánh độ chính xác của mô hình.
+3. Hãy tăng dữ liệu huấn luyện từ nhiều cuốn sách. 
+3. Tăng dữ liệu huấn luyện bao gồm nhiều cuốn sách. Hãy xem bạn có thể giảm độ hỗn loạn tới mức nào?
```
Comment (lkhphuc) [inline chapter_recurrent-modern/deep-rnn_vn.md:205]:
```suggestion
3. Tăng dữ liệu huấn luyện lên gồm nhiều cuốn sách. Hãy xem bạn có thể giảm độ hỗn loạn tới mức nào?
```
